src/uu/cp/src/cp.rs

```diff
@@ -866,6 +871,55 @@
     }
 }
 
+pub fn normalize_path<P: AsRef<Path>>(path: P) -> PathBuf {
```

Comment:
After looking at this function again, I realized it is *very* similar to `uucore::fs::resolve_relative_path()` (especially when viewed in context).  Perhaps it would make more sense to use that?

Reply:
well spotted!
It works locally, I pushed an update to see if the CI is happy too